### PR TITLE
incomplete fb login button (not visible, no styles, why)

### PR DESCRIPTION
**Pull request by imagdalene**
not working but must merge to dev to prevent stale branches, will resume April 2

### CODE REVIEW

**Comment by imagdalene** [comment]
The FB Button mounts correctly but there is no style and nothing clickable

Merging this to prevent stale branches on my side

File: src/components/card.js [hunk 1]
```diff
@@ -4,10 +4,11 @@ import { violet } from '../constants'
 const Card = ({
   headerContent,
   headerDescContent,
-  imgAsset,
+  // imgAsset,
```
Comment (knardi) [inline src/components/card.js:7]: ? Maybe delete if it's not necessary, or add a comment explaining why.

File: src/components/financialCard.js [hunk 2]
```diff
@@ -1,19 +1,26 @@
-import PropTypes from 'prop-types'
-import React from 'react'
+import React, { useState } from 'react'
+// import PropTypes from 'prop-types'
```
Comment (knardi) [inline src/components/financialCard.js:2]: Same here. Maybe delete or add a comment explaining why it's commented out.

File: src/components/financialCard.js [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
 import { FormattedMessage } from 'react-intl'
+import FbSignIn from './fbSignin'
```
Comment (knardi) [inline src/components/financialCard.js:4]: nit: Different case, capital `I` vs. lowercase `i`

File: src/components/fbSignin.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  usePortal: true,
+}
+
+export default ({ show }) => {
+  return (
+    <Drawer {...defaultDrawerProps} isOpen={show}>
+      <div className={Classes.DRAWER_BODY}>
+        <div className="container m-auto flex justify-between">
+          <div
+            className="fb-login-button"
```
Comment (knardi) [inline src/components/fbSignin.js:21]: ubernit: I like "fb login button" better as a name for this whole component than "fb sign in". Would prefer `fbLoginButton.js`.

File: src/components/card.js (hunk 1 above)
Comment (imagdalene) [reply src/components/card.js:7]: the original author bypassed commit hooks to include them for a reason

File: src/components/financialCard.js [hunk 5]
```diff
@@ -1,19 +1,26 @@
...
 
 import Card from './card'
 const FiancialCard = () => {
+  const [fbSigninVisible, setFBSigninVisible] = useState(false)
```
Comment (knardi) [inline src/components/financialCard.js:8]: ubernit: Seeing `fb` and `FB` on the same line makes me wonder if we should just spell out `Facebook`

File: src/components/card.js (hunk 1 above)
Comment (knardi) [reply src/components/card.js:7]: @danielsamfdo Should this be deleted?

File: src/components/financialCard.js (hunk 2 above)
Comment (knardi) [reply src/components/financialCard.js:2]: @danielsamfdo Should this be deleted?

**Pull request merged**